src/Extensions/ServiceFabricGuest.Abstractions/IServiceFabricClientWrapper.cs

```diff
@@ -1,16 +1,16 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license.
-
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
 using System.Threading.Tasks;
 using Microsoft.ServiceFabric.Client;
 
 namespace Microsoft.Omex.Extensions.ServiceFabricGuest.Abstractions
-{
+{
 	/// <summary>
 	/// Wrapper for Service Fabric Client
 	/// </summary>
 	public interface IServiceFabricClientWrapper
-	{
+	{

```

Comment:
why are there two changes here? Can you rollback them?

Reply:
I believe that they are after code formatting (probably I forgot about it in previous PR and that's why now it is formatted)

Reply:
IMO: It looks like a small change that does not affect anything at the moment.
But in someday, if this PR gets a look again, and see these redundant changes, you maybe unhappy.